packages/babel-plugin-transform-template-literals/src/index.js

```diff
@@ -3,10 +3,14 @@ import { template, types as t } from "@babel/core";
 
 export default declare((api, options) => {
   api.assertVersion(7);
-  const { loose } = options;
+
+  const ignoreToPrimitiveHint =
+    options.loose || api.assumption("ignoreToPrimitiveHint");
```

Comment:
This check `||` is different than https://github.com/babel/babel/pull/12407#discussion_r536511944.